### DISCUSSION TITLE
UI for beatmap selection filter

### DISCUSSION THREAD

**Discussion by SensouShin** [ppy/osu#17494] [Ideas] [answered]
There is two part:

1. We are living in 2k22 we shouldn't have to use text based filters (even though i like it). It can help new players. I'm pretty sure there are plenty of new  players who doesn't aware of this feature at all because it's somewhat hidden.
2. Ability to save filters. I have found on github many similar requests about collections where you can make them, based on parameters but as a filter it's more flexible and it doesn't require any update if new beatmap present, you can just apply it.

something like this:
![lazer_beatmap_filter](https://user-images.githubusercontent.com/6898013/160299150-215abb15-f67c-4fea-a470-d85e756213a3.png)

**Comment by peppy** [comment, marked as answer]
The filters available will be similar to what you can find on the beatmap listing overlay. A new song select design is coming which will be the first step towards addressing this.

Also see https://github.com/ppy/osu/discussions/17339

**Reply by SensouShin** [reply]
For the first point it sounds good, if it will be expanded compared to beatmaplisting, but what about the second point?

**Reply by Acrido** [reply]
Hey, any updates on filters saving? It hasn't been answered clearly here, if not at all, and it would be a nice QoL feature.

**Reply by smoogipoo** [reply]
There hasn't been any work on it yet, no.
